packages/divider-curve/README.md

```diff
@@ -0,0 +1,25 @@
+# @littlespoon/divider-curve
```

Comment:
Can you rename this component to `Divider`?

Reply:
noticed we use lowercase for `button` - should I use proper case going forward?

Reply:
Sorry for not clarifying, can you rename `packages/divider-curve` to `package/divider` and the component name `DividerCurve` to `Divider` and package.json name `@littlespoon/divider-curve` to `@littlespoon/divider`

Reply:
Update package name and instances of package name to `divider`